yargs.js

```diff
@@ -483,6 +483,12 @@ function Yargs (processArgs, cwd, parentRequire) {
     return self
   }
 
+  self.onFinishCommand = function (f) {
+    argsert('<function>', [f], arguments.length)
+    self._onFinishCommand = f
```

Comment:
Shouldn't _onFinishCommand be freezed/unfreezed?

Reply:
Done, thx